src/mame/galaxian/galaxian.cpp

```diff
@@ -2537,6 +2537,27 @@ void galaxian_state::jumpbug_map(address_map &map)
 	map(0xb000, 0xbfff).r(FUNC(galaxian_state::jumpbug_protection_r));
 }
 
+// Same as 'jumpbug_map' but without the AY-8910
```

Comment:
If it's just that, then you might be able to update the jumpbug_map to call this one and just have the AY methods in there.

something like:
void galaxian_state::jumpbug_map(address_map &map)
{
     jumpbugbrf_map(map);

     AY methods here
}

Reply:
Done!! Thanks!